src/Microsoft.OData.Edm/Validation/EdmError.cs

```diff
@@ -56,4 +75,4 @@ public override string ToString()
             return Convert.ToString(this.ErrorCode, CultureInfo.InvariantCulture) + " : " + this.ErrorMessage;

```

Comment:
@wachugamaina Might you need to take `ErrorLevel` into account when converting the `EdmError` into a string? Especially in those cases where the value is not null

Reply:
Yes, we did consider this, but I did not want to change the output of the to String function, it may break some users code downstream.